feasibility-dsf-process/src/main/java/de/medizininformatik_initiative/feasibility_dsf_process/service/ObfuscateEvaluationResult.java

```diff
@@ -35,11 +41,32 @@ protected void doExecute(DelegateExecution execution, Variables variables) {
     }
 
     private MeasureReport obfuscateFeasibilityCount(MeasureReport measureReport) {
-        var obfuscatedFeasibilityCount = obfuscator.obfuscate(measureReport.getGroupFirstRep().
-                getPopulationFirstRep().getCount());
+        var obfuscatedFeasibilityCount = obfuscator.obfuscate(extractInitialPopulation(measureReport).getCount());
+
+        var obfuscatedMeasureReport = new MeasureReport()
+                .setMeasure(measureReport.getMeasure())
```

Comment:
Please use the order of properties as defined in the [MeasureReport](http://hl7.org/fhir/R4/measure.html) resource because so its more clear what properties are included and what not.